localisation/00_zuwc_united_localisation.csv

```diff
@@ -2981,6 +2980,8 @@ EVTDESC_WCTFT_600;Today [From.GetTitledFirstNameWithNick] announced that [From.G
 bloodline_effect_scarlet_king;Can �GVassalize�! the �YScarlet Crusade�!;;;;;;;;;;;;;x
 vassalize_d_scarlet_crusade;Vassalize the Scarlet Crusade;;;;;;;;;;;;;x
 vassalize_d_scarlet_crusade_desc;As a long-standing ally of the Scarlet Crusade, they could be convinced to become vassals of the realm if they see us fit.;;;;;;;;;;;;;x
+swarm_culture_modifier;Swarm Culture Modifier;;;;;;;;;;;;;x
+numerous_cowards_culture_modifier;Numerous Cowards Culture Modifier;;;;;;;;;;;;;x
```

Comment:
> Numerous Cowards Culture

Imo, that sounds really bad. Idk what to suggest, but I would like to use something more neutral.

Reply:
> > Numerous Cowards Culture
> 
> Imo, that sounds really bad. Idk what to suggest, but I would like to use something more neutral.

Quantity Culture Modifier?

Reply:
Sounds better

Reply:
Done in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1424/commits/8a155e5992521f1ff570d59b83a571ae90786d82.